openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -338,3 +348,58 @@ def get(self, request, course_key_string, *args, **kwargs):
             pass
 
         return Response(resp)
+
+
+class Celebration(DeveloperErrorViewMixin, APIView):
+    """
+    **Use Cases**
+
+        Marks a particular celebration as complete
+
+    **Example Requests**
+
+        POST /api/courseware/celebration/{course_key}
+
+    **Request Parameters**
+
+        Body consists of the following fields:
+
+            * first_section (bool): when a user finishes their first section of a course
```

Comment:
Nit: `when a user finishes their first section of a course` -> `if a celebration should be shown when a user finishes their first section of a course`
The way I read it, the `when` made me think it would be a datetime. I know it's a bool, but just my weirdness

Reply:
Done.